packages/suite/src/components/suite/FormattedCryptoAmount.tsx

```diff
@@ -90,7 +90,7 @@ export const FormattedCryptoAmount = ({
         <Container className={className}>
             {!!signValue && <Sign value={signValue} />}
             <Value data-testid={dataTest}>{formattedValue}</Value>
-            {formattedSymbol && <BlurUrls text={'\u00A0' + formattedSymbol} />}
+            {formattedSymbol && <BlurUrls text={' ' + formattedSymbol} />}
```

Comment:
```suggestion
            {formattedSymbol && <>
              {' '}
              <BlurUrls text={formattedSymbol} />
            </>}
```
Something like that would look better